server/service/src/sync/translations/requisition_line.rs

```diff
@@ -216,4 +227,45 @@ mod tests {
             assert_eq!(translation_result, record.translated_record);
         }
     }
+
+    #[actix_rt::test]
+    async fn test_requisition_line_push_merged() {
+        // The item_links_merged function will merge ALL items into item_a, so all stock_lines should have an item_id of "item_a" regardless of their original item_id.
+        let (_, connection, _, _) = setup_all(
```

Comment:
Nit: looks a bit fragile too me, e.g. if you add more mock data that will break this test. But I might be wrong...

Reply:
So long as no one removes item_a then the test should work. Is a kinda wild approach though haha.